### PR TITLE
fix registration screen

### PR DESCRIPTION
**Pull request by Hiroki0613**
I fixed the registration screen.

iPhone7
https://youtu.be/BnBCoaFClsI

iPhone11Pro (simulator)
https://youtu.be/Vd_of9FiiZI

### CODE REVIEW

File: creator/lib/screens/home_screen.dart [hunk 1]
```diff
@@ -9,6 +11,38 @@ class HomeScreen extends StatelessWidget {
       appBar: AppBar(
         backgroundColor: Colors.green,
         title: Text('ホーム'),
+        iconTheme: IconThemeData(color: Colors.black),
+      ),
+      drawer: Drawer(
```
Comment (dragonfly91) [inline creator/lib/screens/home_screen.dart]: this function is too long now. please break it down

File: creator/lib/screens/home_screen.dart [hunk 2]
```diff
@@ -9,6 +11,38 @@ class HomeScreen extends StatelessWidget {
...
+        child: ListView(
+          children: [
+            DrawerHeader(
+              child: Text(
+                "testPage",
+                style: TextStyle(
+                  fontSize: 24,
+                  color: Colors.white,
+                ),
```
Comment (dragonfly91) [inline creator/lib/screens/home_screen.dart]: This can be just a Container()

File: creator/lib/screens/salon_application_screen.dart [hunk 3]
```diff
@@ -28,20 +30,53 @@ class _SalonApplicationScreenState extends State<SalonApplicationScreen> {
...
+        ),
+      ),
       body: SafeArea(
         child: Container(
           padding: EdgeInsets.symmetric(horizontal: screenWidth * 0.03),
           child: Center(
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
+            child: ListView(
+              // mainAxisAlignment: MainAxisAlignment.center,
```
Comment (dragonfly91) [inline creator/lib/screens/salon_application_screen.dart]: Please remove commented out code

File: creator/lib/screens/salon_application_screen.dart [hunk 4]
```diff
@@ -95,17 +125,44 @@ class _SalonApplicationScreenState extends State<SalonApplicationScreen> {
...
+        keyboardType: keyboardType,
+        decoration: InputDecoration(
+            contentPadding: EdgeInsets.fromLTRB(16, 8, 16, 8),
+            focusedBorder: OutlineInputBorder(
+              borderSide: BorderSide(color: Colors.black, width: 0.5),
+            ),
+            enabledBorder: OutlineInputBorder(
+              borderSide: BorderSide(color: Colors.black, width: 0.5),
+            ),
+            hintText: label),
```
Comment (dragonfly91) [inline creator/lib/screens/salon_application_screen.dart]:
```suggestion
            hintText: label,),
```

**Review by dragonfly91 — APPROVED** [review]
Minor changes needed. Please address them!

File: creator/lib/screens/salon_application_screen.dart (hunk 3 above)
Comment (Hiroki0613) [reply creator/lib/screens/salon_application_screen.dart]: I remove the commented out code.

File: creator/lib/screens/salon_application_screen.dart (hunk 4 above)
Comment (Hiroki0613) [reply creator/lib/screens/salon_application_screen.dart]: I put comma after label

File: creator/lib/screens/home_screen.dart (hunk 1 above)
Comment (Hiroki0613) [reply creator/lib/screens/home_screen.dart]: I break it down to separate method.

**Review by dragonfly91 — APPROVED** [review]
LGTM

**Pull request merged**